deel/torchlip/modules/module.py

```diff
@@ -106,21 +107,28 @@ def __init__(
                     )
                 )
 
-    def vanilla_export(self):
+    def vanilla_export(self, keep_names: bool = False):
```

Comment:
As suggested, you can remove this flag and set the layer names as default behaviour